selfdrive/car/hyundai/interface.py

```diff
@@ -41,6 +41,11 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), car_fw=[], experi
         # non-HDA2
         if 0x1cf not in fingerprint[4]:
           ret.flags |= HyundaiFlags.CANFD_ALT_BUTTONS.value
+        # 2021 Santa Cruz does not have 0x130; GEARS message on 0x40
+        if 0x130 not in fingerprint[4]:
+          ret.flags |= HyundaiFlags.CANFD_ALT_GEARS.value
+      if candidate in CAMERA_SCC_CAR:
```

Comment:
What if this were 'if candidate in CAMERA_SCC_CAR and not (CP.flags & HyundaiFlags.CANFD_HDA2):'?
This way you could add cars like the EV6, I5 and GV60 to the CAMERA_SCC_CAR category, but the hda2 variants would get excluded from actually being flagged. I do not think there are any cars that have both CAMERA_SCC_CAR and non-CAMERA_SCC_CAR variants where both variants are non-hda2. That would see quite a strange thing to do.

Reply:
The 2022 Genesis GV70 being supported in this PR does not do SCC at the camera (0x1A0 on bus 4).

Reply:
Exactly the point, so it would not get put in the SCC Camera Category and wouldn't affected. I did ention the GV60 which I believe operates the same as the EV6/I5.

Reply:
This would be useful for other ICE models like the 2023 Kia Telluride that will have both hda1 and hda2 options.